daisy/logger.go

```diff
@@ -107,11 +109,13 @@ type logEntry struct {
 	StepName       string    `json:"stepName,omitempty"`
 	StepType       string    `json:"stepType,omitempty"`
 	Message        string    `json:"message"`
+
+	Type string `json:"type"`
```

Comment:
This field doesn't seem to be ever used. It's just set to "Daisy" every time. Do you intend to use it in the future or something? Or am I just missing where it's used?

Reply:
These are fields that get sent to cloud logging, most don't get used directly in Daisy